spec/index.html

```diff
@@ -145,17 +171,32 @@ <h3>IRIs</h3>
       <h3>RDF Literals</h3>
 
       <p><a data-cite="RDF12-CONCEPTS#literal">Literals</a>
-        are used to identify values such as strings, numbers,
-        dates.</p>
-
-      <p>
-        Literals (Grammar production <a href="#grammar-production-literal">Literal</a>) have a lexical form followed by a language tag, a datatype IRI, or neither.
-        The representation of the lexical form consists of an
-        initial delimiter <code>&quot;</code> (<span class="codepoint">U+0022</span>), a sequence of permitted
-        characters or numeric escape sequence or string escape sequence, and a final delimiter. Literals may not contain the characters <code>&quot;</code>, <code title="LINE FEED"><sub>LF</sub></code>, <code title="CARRIAGE RETURN"><sub>CR</sub></code> except in their escaped forms. In addition '<code>\</code>' (<span class="codepoint">U+005C</span>) may not appear in any quoted literal except as part of an escape sequence.
-        The corresponding <a data-cite="RDF12-CONCEPTS#lexical-form">RDF lexical form</a> is the characters between the delimiters, after processing any escape sequences.
-        If present, the <a data-cite="RDF12-CONCEPTS#language-tag">language tag</a> is preceded by a '<code>@</code>' (<span class="codepoint">U+0040</span>).
-        If there is no language tag, there may be a <a data-cite="RDF12-CONCEPTS#datatype-iri">datatype IRI</a>, preceded by '<code>^^</code>' (<span class="codepoint">U+005E</span> <span class="codepoint">U+005E</span>). If there is no datatype IRI and no language tag it is a <a data-cite="RDF12-CONCEPTS#simple-literal">simple literal</a> and the datatype is <code>http://www.w3.org/2001/XMLSchema#string</code>.
+        are used to identify values such as strings, numbers, dates.</p>
+
+      <p>Literals (Grammar production <a href="#grammar-production-literal">Literal</a>)
+        have a lexical form followed by a language tag, a datatype IRI, or neither.</p>
+
+      <p>The representation of the lexical form consists of an
+        initial delimiter <code>&quot;</code> (<span class="codepoint">U+0022</span>),
+        a sequence of permitted characters or numeric escape sequence or string escape sequence,
+        and a final delimiter.</p>
+
+      <p>Literals may not contain the characters <code>&quot;</code>,
+        <code title="LINE FEED"><sub>LF</sub></code> (<span class="codepoint">U+000A</span>), or
+        <code title="CARRIAGE RETURN"><sub>CR</sub></code> (<span class="codepoint">U+000D</span>)
+        except in their escaped forms.
+        In addition '<code>\</code>' (<span class="codepoint">U+005C</span>)
+        may not appear in any quoted literal except as part of an escape sequence.</p>
```

Comment:
The Turtle text is imprecise on escaping : w3c/rdf-turtle#5

Add:
```suggestion
        may not appear in any quoted literal except as part of an escape sequence
        and a <code>"</code> (<span class="codepoint">U+0022</span>) character
        can only be included in a quote literal using an escape sequence.
        </p>
```
It's not necessary but if one spec (Turtle) mentions it, the others ought to.